src/main/java/bio/terra/stairway/FlightDao.java

```diff
@@ -159,6 +150,35 @@ private String lookupStairwayInstanceQuery(Connection connection, String stairwa
     }
   }
 
+  /**
+   * This call can be used in conjunction with a list of the currently active stairway instances to
+   * drive recovery of orphaned flights.
+   *
+   * @return list of the stairway instances known to stairway
+   * @throws DatabaseOperationException
+   */
+  List<String> getStairwayInstanceList() throws DatabaseOperationException {
+    final String sql = "SELECT stairway_name FROM " + STAIRWAY_INSTANCE_TABLE;
+    List<String> instanceList = new ArrayList<>();
+
+    try (Connection connection = dataSource.getConnection()) {
+      startReadOnlyTransaction(connection);
+
+      try (NamedParameterPreparedStatement instanceStatement =
+              new NamedParameterPreparedStatement(connection, sql);
+          ResultSet rs = instanceStatement.getPreparedStatement().executeQuery()) {
```

Comment:
minor nit: weird indentation here?

Reply:
This project uses the google java formatting and the idea plugin for same. I've noticed a few oddities.

I reformatted, but the formatter may switch it back: it auto-formats on build.